src/main/java/ch/njol/skript/lang/globals/GlobalOptions.java

```diff
@@ -0,0 +1,40 @@
+package ch.njol.skript.lang.globals;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.config.Config;
+import ch.njol.skript.lang.OptionRegistry;
+
+import java.io.IOException;
+import java.util.*;
+
+/**
+ * Represents the 'globals/options.sk' file
+ */
+public class GlobalOptions extends GlobalFile {
+
+	public GlobalOptions() {
+		super(Skript.getAddonInstance(), "options");
+	}
+
+	/**
+	 * Loads the 'globals/options.sk' file
+	 */
+	@Override
+	public void load() {
+		try {
+			Config config = new Config(file, true, false, ":");
+			config.getMainNode().convertToEntries(-1);
+			OptionRegistry optionRegistry = OptionRegistry.get();
+			optionRegistry.loadGlobalOptions(config.getMainNode());
+
+			// for unit tests
+			if (Skript.testing()) {
```

Comment:
I might prefer this to be done through a JUnit or something. Or maybe have a test-only effect to register a global option.